katharsis-servlet/src/test/java/io/katharsis/servlet/resource/repository/LocaleRepository.java

```diff
@@ -0,0 +1,42 @@
+package io.katharsis.servlet.resource.repository;
+
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import io.katharsis.queryParams.QueryParams;
+import io.katharsis.repository.ResourceRepository;
+import io.katharsis.servlet.resource.model.Locale;
+
+/**
+ * Created by nickmitchell on 1/6/17.
+ */
+public class LocaleRepository implements ResourceRepository<Locale, Long> {
```

Comment:
I don't understand what the locale has to do with the serializer and why there is a repository